win32/wsyslog.c

```diff
@@ -95,10 +95,6 @@ void vsyslog(int priority, const char *message, va_list args)
 	DWORD evid;
 	wchar_t *strsw[2];
 
-	/* default event source */
```

Comment:
There are no calls to `vsyslog()` (directly), it all flows through `php_syslog()` so there's no way to reach here without `openlog()` having been called first (and thereby setting `log_source`).